server/integration-tests/calabash-test-suite/features/steps/shared.rb

```diff
@@ -14,8 +14,9 @@
 
 Then(/^I print it to stdout$/) do
   s = query("edittext", :getText).first.gsub("URI: //ready\nparams: {json={}\n}", "").gsub("URI: //ping\nparams: {json={}\n}", "")
-  puts s
-  $stdout.puts s
+  Kernel.puts s
+  # private method `puts' called for #<Cucumber::Formatter::Interceptor::Pipe:0x00007fe6e2ce40c0> (NoMethodError)
+  # $stdout.puts s
```

Comment:
Should we remove this commented line from the test?

Reply:
@leffsu Do you remember what is this fix for?

Reply:
It is a fix for the line not working. I've commented it at the time but I think it's safe to delete it now.